src/views/Languages/index.jsx

```diff
@@ -291,18 +239,21 @@ export default class Languages extends Component {
                 ...(bug.keywords || []),
                 ...extractWhiteboardTags(bug.whiteboard),
               ],
-              summary: bug.summary,
+              summary: {
+                title: bug.summary,
+                url: `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.id}`,
+              },
               lastUpdated: bug.lastChanged,
               id: bug.id,
-              url: `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.id}`,
             })),
-          'summary'
-        )) ||
-      [];
-    const mentoredBugs =
-      (bugzillaData &&
-        bugzillaData.mentored &&
-        uniqBy(
+          'summary.title'
+        );
+
+        ({ hasNextPage } = bugzillaData.goodFirst.pageInfo);
+      }
+
+      if (bugzillaData.mentored) {
+        mentoredBugs = uniqBy(
```

Comment:
can this be as below? thus the if is no longer needed. Also, can we do this in the render method. Thus, we don't need to keep the items state.
```suggestion
    const mentoredBugs =
      (bugzillaData &&
        bugzillaData.mentored &&
        uniqBy(
```